File: Swift-compiling/Stdinout.swift

func Stdinout() {

	print("こんにちは。私の名前はSwift,即ちアマツバメです。")
	print("あなたのことを教えてください")
	let aboutYou = readLine() ?? ""
	if aboutYou != "" {
		print("あなたは \(aboutYou) ですね。わかりました。")
	}
	else {
		print("あなたのことがよくわかりませんでした。")
	}

}

File: Scala-compiling/Stdinout.scala
object Stdinout {

    def main(args: Array[String]):Unit = {

        println("こんにちは。私の名前はScala。")
        println("あなたのことを教えてください")
        val aboutYou = io.StdIn.readLine("あなたは...")
        println(s"あなたは $aboutYou ですね。わかりました。")

        println("\r\n")

    }

}